### ISSUE TITLE
Auto capture Performance, Navigation and Resource timings

### ISSUE THREAD

**Issue by codecapitano** [grafana/faro-web-sdk#112]
## Description

For APM it is important to able track Performance, Navigation and Resource Timings to drill down into that data to for example find slow page loads, request times we can improved have an idea about the size of assets and many many more.

Having that data also enables functionality like adding alerts base on that data, define and observe performance and many more.

## Proposed solution
Add and instrumentation which automatically collects and sends that data. 

## Context
[See: Navigation and resource timings](https://developer.mozilla.org/en-US/docs/Web/Performance/Navigation_and_resource_timings#performance_entry_api)

### TL;DR Some Perf. Timings we can collect or calculate based on browser metrics
![image](https://user-images.githubusercontent.com/47627413/218040177-4501db2c-a65e-4a9f-b7b8-c3e1737434b2.png)

### TL;DR Some  Resource Timings we can collect or calculate based on browser metrics
![image](https://user-images.githubusercontent.com/47627413/218040274-602ea580-e964-4d21-a578-09b657cd49a3.png)
